tsdef/LocalDataTrack.d.ts

```diff
@@ -1,4 +1,4 @@
-import { LocalDataTrackOptions } from './LocalDataTrackOptions';
+import { LocalDataTrackOptions } from './types';
```

Comment:
This will form a circular import. LocalDataTrackOptions is imported from `types` to which `LocalDataTrack` is exported to, which imports LocalDataTrackOptions from `types`, and so on.

What we need to do is:

* Create `LocalTrackOptions.d.ts`
* Retain `LocalDataTrackOptions.d.ts`
* Derive LocalAudioTrackOptions, LocalVideoTrackOptions and LocalDataTrackOptions from `LocalTrackOptions.d.ts`
* In `types.d.ts`, export LocalTrackOptions as follows:
```ts
export { LocalTrackOptions } from './LocalTrackOptions';
```